inference-engine/tests/functional/plugin/cpu/single_layer_tests/split.cpp

```diff
@@ -91,8 +91,11 @@ const auto planar_5D_ref = CPUSpecificParams{{ncdhw}, {ncdhw}, {"ref"}, "ref"};
 const auto planar_4D = CPUSpecificParams{{nchw}, {nchw}, {}, "unknown"};
 const auto planar_5D = CPUSpecificParams{{ncdhw}, {ncdhw}, {}, "unknown"};
 
-const auto planarChannels_4D = CPUSpecificParams{{nhwc}, {nhwc}, {}, "ref"};
-const auto planarChannels_5D = CPUSpecificParams{{ndhwc}, {ndhwc}, {}, "ref"};
+const auto perChannels_4D = CPUSpecificParams{{nhwc}, {nhwc}, {}, "ref"};
+const auto perChannels_5D = CPUSpecificParams{{ndhwc}, {ndhwc}, {}, "ref"};
+
+const auto planarChannels_4D = CPUSpecificParams{{nhwc}, {nchw}, {}, "ref"};
```

Comment:
Maybe `perChannelsToPlanar_4D` ?
I think that `planarChannels_4D` doesn't say anything

Reply:
Yes, certainly. Done ca721fa.